infoblox_client/objects.py

```diff
@@ -352,18 +352,27 @@ def __repr__(self):
                    for name in self._ea_dict)
         return "EAs:{0}".format(','.join(eas))
 
+    @staticmethod
+    def _value_to_bool(value):
```

Comment:
This is the ENUM as string values we added when creating a EA definition.
The string has to match exactly what we typed in EA definition.
For example, 'True' is not the same as 'true'.
So the current implementation satisfies this requirement.